build_docs.py

```diff
@@ -80,11 +95,57 @@ def changed_files(directory, other):
     return changed
 
 
+def git_clone(repository, directory, branch='master'):
+    """Clone or update the given branch of the given repository in the
+    given directory.
+    """
+    logging.info("Updating repository %s in %s", repository, directory)
+    try:
+        shell_out("git -C {} checkout {}".format(directory, branch))
```

Comment:
Oh, wait, the "{}" or "%s" in shell commands are not new: the current code already use them. You can ignore my comment. It can be enhanced in a different PR ;-)